config/default

```diff
@@ -1,15 +1,11 @@
-export instanceid = main
-export base_url = /
-
-export debug_port = 6553
+export instanceid ?= main
+export base_url ?= /
+export debug_port ?= 6553
 
 # the size of the connection pool for http requests (to the API)
 export http_request_connection_pool_size = 200
 
-host = c2corgv6.demo-camptocamp.com
-url = http://$(host)
-
-export api_url = http://c2corgv6-api.demo-camptocamp.com
+export api_url = http://localhost:6543
```

Comment:
Are you sure of this?
``api_url`` is the "external" URL, it cannot be "localhost" by default.
The former value looks more relevant and consistent with the params below.